src/core_plugins/kibana/public/home/home.less

```diff
@@ -0,0 +1,52 @@
+@import (reference) "~ui/styles/variables.less";
+
+.home {
+  background-color: @globalColorLightestGray;
+  min-height: 100vh;
+
+  .kuiPanelBody {
```

Comment:
Can we specify custom class names, e.g. `.homePanel` to create these "overrides"? This way we don't couple these selectors to the classes used in the UI Framework. If possible, I think we should try to avoid referring to UI Framework classes in any of our selectors.